Fix emotion label not being applied to components with numbers in name (#2122)
emotion-js/emotion: .changeset/nasty-starfishes-retire.md

---
'@emotion/react': patch
---

> Fix emotion label not being applied to components with numbers in name

Reviewer: ```suggestion
Improved label extraction from the stack traces. We should be able to extract labels from components with numbers in their names now.
```

> Improved label extraction from the stack traces. We should be able to extract labels from components with numbers in their names now.

Reviewer: ```suggestion
Improved label extraction from the stack traces to handle components with numbers in their names.
```
